README.md

```diff
@@ -197,3 +197,26 @@ your openstack Cluster API Kubernetes cluster.
    ```bash
    rm -rf $HOME/.ssh/openstack_tmp*
    ```
+
+### Trouble shooting
+
+This guide (based on minikube and others should be similar) explains general info on how to debug issues if cluster failed to create.
+
+1. Get openstack contain name, the output depends on the system you are running.
+   the `minikube.kubeconfig` which is bootstrap cluster's kubeconfig by default locates at `cmd/clusterctl` folder.
+
+   ```
+   # kubectl --kubeconfig minikube.kubeconfig get pods --all-namespaces
+   NAMESPACE                   NAME                                     READY   STATUS    RESTARTS   AGE
+   .....
+   openstack-provider-system   clusterapi-controllers-xxxxxxxxx-xxxxx   1/1     Running   0          27m
+   system                      controller-manager-0                     1/1     Running   0          27m
+   ```
+
+2. Get log of clusterapi-controllers-xxxxxxxx-xxxxx
+
+   ```
+   # kubectl --kubeconfig minikube.kubeconfig log clusterapi-controllers-xxxxxxxxx-xxxxx -n openstack-provider-system
+   ```
+
+3. If above log is not enough, you might want check openstack log as well, e.g. `/var/log/nova/nova-api.log` and `/var/log/nova/nova-compute.log` (by default) if create machine has some issue.
```

Comment:
Sometimes, the end user may not have permission to check those logs of nova. Can we remove this and only keep the controller log?